api/types/client.go

```diff
@@ -96,6 +96,7 @@ type SiteConfigSpec struct {
 	Router              RouterOptions
 	Controller          ControllerOptions
 	ConfigSync          ConfigSyncOptions
+	LoadBalancerTimeout int
```

Comment:
This is not needed any more is that right?

Reply:
Correct! it is removed now, thanks!